scripts/find-packages-on-old-config.js

```diff
@@ -2,10 +2,10 @@
 const fs = require('fs');
 const path = require('path');
 
-const { forEachPackage } = require('@mongodb-js/monorepo-tools');
+const { listAllPackages } = require('@mongodb-js/monorepo-tools');
 
-forEachPackage((props) => {
+for await (const props of listAllPackages()) {
```

Comment:
This throws 

```
/Users/sergey.petushkov/Projects/MongoDB/compass/scripts/find-packages-on-old-config.js:7
for await (const props of listAllPackages()) {
    ^^^^^

SyntaxError: Unexpected reserved word
```

I guess our version of node doesn't support top level awat or something like that (I'm not sure what exactly)

Don't know if it's worth dealing with if we decided to get rid of forEachPackage, can probably just remove this file

Reply:
Weird. Why does it work for the other script, then? I used it to make the electron bump PR 🤔

Reply:
OH the electron bump one doesn't have it at the top level. I'll just tweak this file.

Reply:
Decided to just drop this file - it finds no more old packages anyway.